composer.json

```diff
@@ -25,8 +25,8 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.4",
-        "squizlabs/php_codesniffer": "^3.1",
+        "phpunit/phpunit": "^7 | ^8 | ^9",
```

Comment:
Just for clarification, is there a reason why we not simply use the latest phpunit version? Are there PHP version constraints?

Reply:
Well, in the CI we test php 7.x versions and not all the versions use the same phpunit version and for a second reason: on Debian we will run phpunit on the latest version packaged and available. Supporting all versions in range of the PHP versions supported is always appreciated by people packaging your library :)

If the support is not as large as we need to run the test suite we need to waste time building patches like this one for example: https://salsa.debian.org/php-team/pear/php-bacon-baconqrcode/-/blob/main/debian/patches/fix-phpunit.patch or https://salsa.debian.org/phpmyadmin-team/phpmyadmin/-/blob/debian/4%254.9.5+dfsg1-2_bpo10+1/debian/patches/phpunit-7.patch